docs/source/endpoints/content-rules.md

```diff
@@ -0,0 +1,169 @@
+---
+myst:
+  html_meta:
+    "description": "Available content rules in a Plone site can be listed and queried by accessing the /@content-rules endpoint on any context. Access requires the ManageContentRules permission."
+    "property=og:description": "Available content rules in a Plone site can be listed and queried by accessing the /@content-rules endpoint on any context. Access requires the ManageContentRules permission."
+    "property=og:title": "Content Rules"
+    "keywords": "Plone, plone.restapi, REST, API, ContentRules, Rules"
+---
+
+(content-rules)=
+
+# Content Rules
+
+```{note}
+These docs are generated by code tests, therefore you will see some `test` rules appear here.
+```
+
+Available content rules in a Plone site can be listed and queried by accessing the `/@content-rules` endpoint in any context.
```

Comment:
Would it be possible to add one or two paragraphs about what content rules are (in general)? We usually start a docs section in REST API by describing the feature. We have to take into account that the REST API documentation is read by people who are not familiar with all the features of Plone. And even if they are, it does not hurt to briefly describe in one or two sentences what we are talking about.

Reply:
A link to https://docs.plone.org/working-with-content/managing-content/contentrules.html would be sufficient for now.

At the same time, a new issue should be created in the `plone/documentation` repo to migrate that source material into Plone 6 docs. It needs a good editor to improve its quality.

Reply:
@stevepiercy we can add a link to the docs. Though, IMO this does not replace a short introduction to what this docs page is about. As said, we tried to do this for all plone.restapi pages and we should make this a requirement for new docs that we create.

Reply:
How's this?

https://github.com/plone/plone.restapi/pull/1496#pullrequestreview-1127573464

Also I created a new issue to migrate the content from 5.2 to 6.0:

https://github.com/plone/documentation/issues/1336